File: pset6/mario/mario.py
from cs50 import get_int

# Valid input acception
while True:
    h = get_int("Height? ")
    if h >= 1 and h <= 8:
        break

# Rows
for i in range(h):
    # Column blank
    for j in range(2, h-i+1):
        print(" ", end="")

    # Column first half
    for j in range(h-i, h+1):
        print("#", end="")

    # Column void in middle
    print("  ", end="")

    # Column secind half
    for j in range(h+2, h+2+i+1):
        print("#", end="")

    # Print new line
    print()

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int h, i, j;
    do                                  //valid input acception
    {
        h = get_int("Height?");
    }
    while (h < 1 || h > 8);

    for (i = 1; i <= h; i++)                 //rows
    {
        for (j = 1 ; j <= h - i ; j++)           //columns blank spaces
        {
            printf(" ");
        }

        for (j = h - i + 1 ; j <= h ; j++)         //columns first half pyramid
        {
            printf("#");
        }

        printf("  ");                   //columns void spaces

        for (j = h + 3 ; j <= h + 2 + i ; j++)       //columns second half pyramid
        {
            printf("#");
        }

        printf("\n");

    }
    return 0;
}